src/components/map/MapDetail.js

```diff
@@ -135,16 +136,18 @@ const MapDetail = () => {
       </Container>
       <Routes>
         <Route
-          path="/point"
+          path="/invitation"
           element={
-            <Modal size="big">포토를 가져와서 보여주는 모달입니다.</Modal>
+            <Modal size="small">
+              <NewInvitation />
+            </Modal>
           }
         />
         <Route
-          path="/invitation"
+          path="photos"
           element={
-            <Modal size="small">
-              <NewInvitation />
+            <Modal>
```

Comment:
```suggestion
            <Modal size="big">
```
Modal 컴포넌트에 prop으로 size 넘겨주셔야해요..!

Reply:
사이즈 변동 프랍을 추가해주셨네요 :)  PhotoList 모달창 사이즈를 big 수정하겠습니다 ~